io-engine/src/core/snapshot.rs

```diff
@@ -79,6 +83,7 @@ impl SnapshotXattrs {
             Self::TxId => "mayastor.tx_id",
             Self::EntityId => "mayastor.entity_id",
             Self::ParentId => "mayastor.parent_id",
+            Self::SnapshotUuid => "uuid",
```

Comment:
any reason some are mayastor prefixed and others are not?

Reply:
> any reason some are mayastor prefixed and others are not?

I think We can uniform it. Last parameter is without `mayastor` prefix as in SPDK code the xattr name is without prefix.
Other 3 parameters with prefix can be changed back without prefix or last one to change to with prefix both in SPDK code and DP code. @mtzaurus What is your suggestion ?